xclim/indicators/atmos/_conversion.py

```diff
@@ -215,3 +218,28 @@ class Converter(Indicator):
     ),
     compute=wrapped_partial(indices.wind_chill_index, mask_invalid=True),
 )
+
+
+potential_evapotranspiration = Converter(
+    identifier="potential_evapotranspiration",
+    var_name="water_evaporation_flux",
+    units="mm day-1",
+    standard_name="water_evapotranspiration_flux",
```

Comment:
```suggestion
    identifier="potential_evapotranspiration",
    var_name="evspsblpot",
    units="kg m-2 s-1",
    standard_name="water_potential_evaporation_flux",
```

Let's stick to CMIP6 and CF metadata.